src/qw/design_stages/main.py

```diff
@@ -123,6 +215,14 @@ def get_remote_stages(service: Service) -> DesignStages:
         # Could have multiple design stages from the same pull request so allow multiple outputs from a single issue
         if "qw-user-need" in issue.labels:
             output_stages.append(UserNeed.from_issue(issue))
-        if "qw-requirement" in issue.labels:
+        elif "qw-requirement" in issue.labels:
             output_stages.append(Requirement.from_issue(issue))
+    for pr in service.pull_requests:
+        if "qw-ignore" in issue.labels:
+            logger.debug(
+                "PR {number} tagged to be ignored, skipping",
+                number=pr.number,
+            )
+            continue
+        output_stages.append(DesignOutput.from_pr(pr))
```

Comment:
I think we should check for tags here as wel, because a PR could be a verification or validation?